pkg/deploy/lattice/service_synthesizer.go

```diff
@@ -65,6 +70,14 @@ func (s *serviceSynthesizer) Synthesize(ctx context.Context) error {
 			s.latticeDataStore.AddLatticeService(resService.Spec.Name, resService.Spec.Namespace,
 				serviceStatus.ServiceARN, serviceStatus.ServiceID, serviceStatus.ServiceDNS)
 
+			glog.V(6).Infof("Creating Service DNS")
+			resService.Status = &serviceStatus
+			err = s.dnsEndpointManager.Create(ctx, resService)
+			if err != nil {
+				glog.V(6).Infof("Failed creating service DNS: %v", err)
```

Comment:
can u make this V(2) for debugging purpose?

Reply:
fixed